Userland/Libraries/LibProtocol/WebSocket.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2021, The SerenityOS developers.
```

Comment:
Please consider adding your personal copyright instead of this generic one. (I won't ask twice, but IMO it's very nice when people put their name on things. :)

Reply:
Ok, sure! I took the liberty of changing the copyright headers for `LibWebSockets` and `telws` in an extra commit, since I guess I'll be using my own name from now on. I hope that's fine!